app/Nova/TravelAssignment.php

```diff
@@ -93,6 +101,36 @@ public function fields(NovaRequest $request): array
                             ?->id
                 ),
 
+            Code::make('Matrix Itinerary')
```

Comment:
It's a little awkward how the field defaults to `null` on the TA creation page. Not a deal breaker.

Reply:
Yeah I fiddled around with that but I couldn't get it to be blank.